src/pad_controller.rs

```diff
@@ -9,27 +9,25 @@ use PadActionEntry;
 use PadController;
 
 pub trait PadControllerExtManual: 'static {
-    #[cfg(any(feature = "v3_22", feature = "dox"))]
     fn set_action_entries(&self, entries: &[PadActionEntry]);
```

Comment:
Please also implement https://github.com/gtk-rs/gtk/pull/826#discussion_r293673105 and https://github.com/gtk-rs/gtk/pull/826#discussion_r293673403